client/src/components/App/Content/UsersManagement/UsersManagement.tsx

```diff
@@ -31,9 +31,11 @@ interface CellNameSort {
 
 const usersManagementTitle = 'ניהול משתמשים';
 const sourceOrganizationLabel = 'מסגרת';
+const searchBarLabel = 'הכנס שם או שם משתמש...';
+
 const notActiveSortFields: string[] = [UsersManagementTableHeadersNames.WATCH, UsersManagementTableHeadersNames.LANGUAGES,
-UsersManagementTableHeadersNames.COUNTY, UsersManagementTableHeadersNames.USER_TYPE,
-UsersManagementTableHeadersNames.DESK];
+                                       UsersManagementTableHeadersNames.COUNTY, UsersManagementTableHeadersNames.USER_TYPE,
+                                       UsersManagementTableHeadersNames.DESK];
```

Comment:
if the spacing is long, fix it... not sure if it's git visualization

Reply:
fixed